virocon/plotting.py

```diff
@@ -147,7 +147,7 @@ def ppf(self, q):
         dist_wrapper = MarginalDistWrapper(model, dim)
         ax = axes[dim]
 
-        sts.probplot(sample[:, dim], dist=dist_wrapper, fit=False, plot=ax)
+        sts.probplot(sample[:, dim], dist=dist_wrapper, fit=True, plot=ax)
```

Comment:
As discussed, the fit regression line is not helpful in determining the goodness of fit. Thus we do not want to plot it. Instead we would like to plot a 45 degree line.
Thus we should set fit=False again. 
Then in the following lines check if a line was drawn (scipy version prior to 1.7.0).
If a line was drawn we should remove it.
Afterwards we should draw a 45 degree line instead.